pykechain/models/widgets/widget.py

```diff
@@ -100,9 +100,10 @@ def title_visible(self) -> Optional[str]:
                 elif self.widget_type in [
                     WidgetTypes.ATTACHMENTVIEWER,
                     WidgetTypes.SIGNATURE,
+                    WidgetTypes.WEATHER,
                 ]:
                     return self._client.property(
-                        pk=self.meta.get(AssociatedObjectId.PROPERTY_INSTANCE_ID),
+                        pk=self.meta.get(AssociatedObjectId.PROPERTY_MODEL_ID),
```

Comment:
Similar to the property grid, if the category of the form is INSTANCE, then the default title is based on the property instance.

Reply:
Done, added an if-else statement to check if the self.meta.get(AssociatedObjectId.PROPERTY_INSTANCE_ID) exists first